src/display.c

```diff
@@ -716,6 +716,14 @@ PyImaging_EventLoopWin32(PyObject *self, PyObject *args) {
 
 #define GET32(p, o) ((DWORD *)(p + o))[0]
 
+int
```

Comment:
```suggestion
int CALLBACK
```
You need to specify the `__stdcall` call convention for 32-bit builds as the default is `__cdecl`.
It worked on 64-bit because there is only a single call convention used there.

I'm not sure why this is missing from the documentation, I had to look at `wingdi.h` to figure this out.

Reply:
Thanks for the thoughts, but I can't find a way to put them together into something that passes on 32-bit. Would you be able to create a PR?

Reply:
https://github.com/radarhere/Pillow/pull/28 fixed this.